src/sql/parts/notebook/notebook.component.ts

```diff
@@ -371,6 +372,8 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		let addTextCellButton = new AddCellAction('notebook.AddTextCell', localize('text', 'Text'), 'notebook-button icon-add');
 		addTextCellButton.cellType = CellTypes.Markdown;
 
+		let clearResultsButton = new ClearAllOutputsAction('notebook.ClearAllOutputs', localize('clearResults', 'Clear Results'), 'notebook-button icon-clear-results');
```

Comment:
we should be consistent with naming. Please consider renaming ClearAllOutputsAction to ClearAllResultsAction

Reply:
Agree from a consistency point of view. So far, we have been follwing the convention of defining code cell's result as 'Output' across our code base. Looking at output classess, interfaces and variables, I decided to maintain the same naming convension here as well. Howver, the label can be any thing, in this case it happened to be "Clear Results", but from a coding perspective I am more inclined to keep it as 'Output' inline with other output objects. I am happy to change it if it's really needed.